config/routes.rb

```diff
@@ -145,6 +145,9 @@
 
     get '/manage/email/:id' => 'users/emails#edit', as: :manage_email
     match '/manage/email/:id' => 'users/emails#update', via: %i[patch put]
+    delete '/manage/email/:id' => 'users/emails#delete'
```

Comment:
What do you think about putting this route behind the feature flag to make sure this functionality is not available if email additions is disabled.

Reply:
This is done